PovertyInfo/src/interactiveinfo/GetPovertyInfo.java

```diff
@@ -0,0 +1,30 @@
+package interactiveinfo;
+
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class GetPovertyInfo {
+
+	public static String getMatch(String county) throws FileNotFoundException {
```

Comment:
I like that you added the `throws` here. I would say it is more typical to put a `try` and `catch` around the block of code than to put the `throws` at the method level.